scripts/generateAssetData/overrides.ts

```diff
@@ -3,7 +3,7 @@ import type { Asset } from '@shapeshiftoss/types'
 
 const fox: Partial<Asset> = {
   color: '#3761F9',
-  icon: 'https://assets.coincap.io/assets/icons/256/fox.png',
+  icon: 'https://raw.githubusercontent.com/trustwallet/assets/b7a5f12d893fcf58e0eb1dd64478f076857b720b/blockchains/ethereum/assets/0xc770EEfAd204B5180dF6a14Ee197D99d808ee52d/logo.png',
```

Comment:
Can we add a TODO somewhere either in the codebase or as an issue, to tackle this overide?
Even after the new icon is merged in Coingecko, the override will still apply since we're using `b7a5f12d893fcf58e0eb1dd64478f076857b720b` hash

Reply:
Not right anymore because of [198a41e](https://github.com/shapeshift/web/pull/7298/commits/198a41ed65fa7b9baf2131ba2eac05808e263b2a)